### PR TITLE
Refactor repo

### PR DESCRIPTION
**Pull request by ajlee21**
This PR is cleaning up the repository to prepare for manuscript submission.

Main changes include:
1. Updating the Readme
2. Removing and formating files with black

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,58 @@
-# Detecting generic gene expression signals
+# Generic transcriptional responses revealed using _SOPHIE_: Specific cOntext Pattern Highlighting In Expression data
 
-**Alexandra J. Lee, Rani K. Powers, Dallas L. Mould, Dongbo Hu, Georgia Doing, Jake Crawford, James C. Costello, Deborah A. Hogan, Casey S. Greene**
+**Alexandra J. Lee, Dallas L. Mould, Jake Crawford, Dongbo Hu, Rani K. Powers, Georgia Doing, James C. Costello, Deborah A. Hogan, Casey S. Greene**
 
 **University of Pennsylvania, University of Colorado Anschutz Medical Campus, Dartmouth College**
 
-**Rationale**: People performing differential expression (DE) analysis found that some genes and subsequent pathways are more likely to be differentially expressed even across a wide range of experimental designs ([Powers et. al., Bioinformatics 2018](https://academic.oup.com/bioinformatics/article/34/13/i555/5045793 ); [Crow et. al., PNAS 2019](https://www.pnas.org/content/116/13/6491)). 
+<!--- [![PDF Manuscript](https://img.shields.io/badge/manuscript-PDF-blue.svg)](https://academic.oup.com/gigascience/article/9/11/giaa117/5952607) --->
```
Comment (jjc2718) [inline README.md:7]: This looks like a link to your previous GigaScience paper - is this what you want here, or is this a placeholder for when you post a preprint of this work?

File: README.md [hunk 2]
```diff
@@ -1,39 +1,58 @@
...
+If you want to perform a new DE analysis in a different biological **context** (i.e. different organism, tissue, media) then you might not have the curated data available. Switching contexts will require re-curation. Similarly, using a different statistical method will require re-curation.
 
 
-**Challenge**: 
-Current methods, including Powers et. al. and Crow et. al., to identify generic genes and pathways rely on manual curation. This curation effort included collecting a large set of samples with corresponding metadata, processing data and performing DE analysis to get ranked list of genes.
+We introduce a new approach to identify generic patterns that uses generative neural networks to produce a null or background set of transcriptomic experiments.
+Analyzing a target experiment against this automatically generated background set makes it straightforward to separate generic and specific results.
+This approach, called SOPHIE for specific context pattern highlighting in expression data, can be applied to any new platform or species for which there is a large collection of unlabeled gene expression data.
```
Comment (jjc2718) [inline README.md:17]:
```suggestion
This approach, called SOPHIE for Specific cOntext Pattern Highlighting In Expression data, can be applied to any new platform or species for which there is a large collection of unlabeled gene expression data.
```

File: README.md [hunk 3]
```diff
@@ -1,39 +1,58 @@
...
+Here, we apply SOPHIE to the analysis of both human and bacterial datasets, and use this method to highlight the ability to detect highly specific but low magnitude transcriptional signals that are biologically relevant.
+The reusable notebooks for training neural networks and for the use of pre-trained generative models for the analysis of differential expression experiments may be broadly useful for the
+prioritization of specific findings in complex datasets.
```
Comment (jjc2718) [inline README.md:20]:
```suggestion
The reusable notebooks for training neural networks and for the use of pre-trained generative models for the analysis of differential expression experiments may be broadly useful for the prioritization of specific findings in complex datasets.
```
(just merging this sentence to be on a single line)

File: install.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+conda env create --file environment.yml
```
Comment (jjc2718) [inline install.sh:1]: Probably good to add a header indicating this is a `bash` script (see, e.g. https://stackoverflow.com/q/8967902).

**Review by jjc2718 — APPROVED** [review]
Looks good - just a few small suggestions.

File: README.md (hunk 1 above)
Comment (ajlee21) [reply README.md:7]: yes this is just a placeholder for now to remind me to add this information

**Pull request merged**